src/config.js

```diff
@@ -49,8 +49,15 @@ export const AUTH_TYPE_USERNAME = 'local-contextual-username';
 export const AUTH_TYPE_PASSWORD = 'local-contextual-username-password';
 export const TIMEOUT = 60000;
 
+export const DEFAULT_LANGUAGE = 'en';
+
+export const MODE_STATIC = 'static';
+export const MODE_OFFLINE = 'offline';
```

Comment:
I'm thinking `offline` should be `read-only`, as `static` would also work `offline`. Just a thought. We can discuss next week.